src/HttpClientInterception/HttpRequestInterceptionBuilder.cs

```diff
@@ -122,6 +123,12 @@ public HttpRequestInterceptionBuilder ForQuery(string query)
             return this;
         }
 
+        public HttpRequestInterceptionBuilder IgnoringQuery()
```

Comment:
Personally I think this would be better phrased as a method with a boolean parameter, and optionally with this method as is which sets it to `true`. Otherwise if you turn it on, there's no way to turn it back off.